pilot/pkg/bootstrap/certcontroller.go

```diff
@@ -201,7 +201,8 @@ func (s *Server) watchRootCertAndGenKeyCert(names []string, stop <-chan struct{}
 func (s *Server) initCertificateWatches(tlsOptions TLSOptions) error {
 	hasPluginCert := hasCustomTLSCerts(tlsOptions)
 	// If there is neither plugin cert nor istiod signed cert, return.
```

Comment:
Comment seems out of date? comment suggests `!plugin && !istiod` but code is `!plugin && istiod`?